src/api/json/catalog.json

```diff
@@ -4760,6 +4760,19 @@
       "fileMatch": [".all-contributorsrc"],
       "name": "All Contributors configuration file",
       "url": "https://json.schemastore.org/all-contributors.json"
+    },
+    {
+      "name": "Flatpak",
+      "description": "Flatpak manifest file",
+      "fileMatch": [
```

Comment:
This is way too generic and will result in a lot of false positive file matches. Can you narrow it down a bit with a specific folder name in the globbing pattern, for instance?